lib/katello/tasks/repository.rake

```diff
@@ -51,7 +51,7 @@ namespace :katello do
   end
 
   desc "Correct missing pulp repositories. Specify CONTENT_VIEW=name and LIFECYCLE_ENVIRONMENT=name to narrow repositories.  COMMIT=true to perform operation."
-  task :correct_repositories => ["environment", "check_ping"] do
+  task :correct_repositories => ["dynflow:client", "environment", "check_ping"] do
     puts "All operations will be skipped.  Re-run with COMMIT=true to perform corrections." unless commit?
```

Comment:
'check_ping' already has this prerequisite in master, is this still happening in master?

Reply:
Oh I missed that! It's not, but I thought it didn't always occur. I'll close the PR.